pytket/pytket/passes/auto_rebase.py

```diff
@@ -141,4 +141,7 @@ def auto_squash_pass(gateset: Set[OpType]) -> SquashCustom:
     :return: Squash to target gateset
     :rtype: SquashCustom
     """
+    if gateset == {OpType.Rz, OpType.PhasedX}:
+        return SquashRzPhasedX()
```

Comment:
One more thing: can you change this test to check that {Rz, PhasedX} is included in gateset (instead of strictly equal)?